LP-984 - Autoscore learners

sadan opened the pull request:
- script for auto scoring learners in waiting state

sadan committed:
LP-984
- script for auto scoring learners in waiting state

saad-waqar commented on openassessment/management/commands/autoscore_learners.py:
```diff
@@ -0,0 +1,191 @@
+            options = crit['options']
+            points = list(set([o['points'] for o in options]))
+
+            if len(points) == 3:
```
lets change this to `if len(points)>2 .... else: `

sadan committed:
LP-984
- Updated assessment admin
- Updated autoscore script to get courses which ends in last 24 hours

sadan committed:
LP-984
- Updated script and removed end date check
